### DISCUSSION TITLE
Different solid color for skipped habit

### DISCUSSION THREAD

**Discussion by fuhadabdulla** [iSoron/uhabits#987] [Feature Requests]
Can you please make the skipped habit a solid color , i am not able to differentiate skipped habit and not done habit in the calender widget of habit loop . It will be very helpful

**Comment by DelphineM** [comment]
This is exactly the enhancement I was going to ask for.
It seems to be redundant with #615

**Comment by iSoron** [comment]
Hi @skillhacker-code, thank you for the feature request. Could you please add a screenshot, so that the problem becomes more clear?

**Reply by fuhadabdulla** [reply]
Sorry for the late reply , I am adding a screenshot of the calendar widget of Habit Loop . It would be great if you can give us an option to make skipped day of habits a Solid Color .

In the Top widget Weight I am tracking My Exercise (One day Weight Training (I tick that habit) , next Day Cardio(I skip the Habit in cardio day  ) ) 

Thank you so much 

![Screenshot_20210718-140123](https://user-images.githubusercontent.com/68536959/126061038-4c73e770-068f-44fa-b8da-96e63dee44af.png)
